libs/remix-ui/editor/src/lib/suggestion-service/suggestion-service.ts

```diff
@@ -0,0 +1,89 @@
+import EventEmitter from 'events'
+
+export class SuggestionService {
```

Comment:
I see you have another file named `suggestion-service.ts`..
This one or the other should probably be removed.

Reply:
> ah so the solidity copilot in the settings is meant to be replaced?
yes 

> I see you have another file named suggestion-service.ts..
This one or the other should probably be removed. 
I think it is till part of the original  browser model

Reply:
ok, I think it's best to clean up what is no longer used, (copilot in the settings and the duplicate suggestion-service.ts), but perhaps you actually already planned that.